src/mfpbench/lcbench_tabular/benchmark.py

```diff
@@ -178,7 +182,7 @@ def __init__(
         super().__init__(
             table=table,
             name=f"lcbench_tabular-{task_id}",
-            config_name="config_id",
+            config_name="id",
```

Comment:
This is used to extract out the `config_id`'s from the table, this should be `"config_id"` as that's what's in the table. This should not effect `query()` and the fix was something else.